sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -455,3 +455,7 @@ ALTER TABLE `amendments_history` MODIFY `created_time` timestamp NULL COMMENT 'c
 #IfNotColumnTypeDefault batchcom msg_date_sent datetime NULL
 ALTER TABLE `batchcom` MODIFY `msg_date_sent` datetime NULL;
 #EndIf
+
+#IfNotColumnTypeDefault drug_inventory last_notify date NULL
+ALTER TABLE `drug_inventory` MODIFY `last_notify` date NULL;
```

Comment:
1. Also should have something like:
```
UPDATE `drug_inventory` SET `last_notify` = NULL WHERE `last_notify` = '0000-00-00';
```
2. And will need to figure out a way to blank out sql-mode for above statement and then reverting sql-mode back after the above statement